docs/using.rst

```diff
@@ -650,11 +652,11 @@ non-zero exit code. Hooks will only be run if a certificate is due for
 renewal, so you can run the above command frequently via ``certbot renew`` 
 without unnecessarily stopping your webserver.
 
-Unique ``--pre-hook`` and ``--post-hook`` hooks run before and after every renewal
-attempt. If two certificates have identical pre and post hooks, they will be run
+Saved or specified hooks run before and after every renewal attempt.
```

Comment:
I still think this sentence is a little misleading. You covered this same information earlier, so perhaps we should just delete it and adjust the next sentence to say something like:

"If two certificates are due for renewal and have identical pre and post hooks, when `certbot renew` is run, each hook will only be run once."